xivo_dao/resources/endpoint_sip/tests/test_dao.py

```diff
@@ -360,6 +360,31 @@ def test_create_predefined_parameters(self):
             category='user',
         ))
 
+    def test_create_with_different_name_same_username(self):
```

Comment:
What is the purpose of this test? There are no specific code that touch username/name

Reply:
You're right, I added the test as part of the exploration to find where the username was disappearing